joplin/base/models.py

```diff
@@ -1,25 +1,84 @@
 from django.db import models
 
+from modelcluster.contrib.taggit import ClusterTaggableManager
+from modelcluster.fields import ParentalKey
 from modelcluster.models import ClusterableModel
 
-from wagtail.wagtailadmin.edit_handlers import FieldPanel
+from wagtail.api import APIField
+from wagtail.wagtailadmin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.wagtailcore import blocks
+from wagtail.wagtailcore.fields import StreamField, RichTextField
 from wagtail.wagtailcore.models import Page
+from wagtail.wagtailsnippets.blocks import SnippetChooserBlock
 from wagtail.wagtailsnippets.models import register_snippet
 
-
-from wagtail.api import APIField
+from taggit.models import TaggedItemBase
 
 
 class HomePage(Page):
     pass
 
 
+WYSIWYG_FEATURES = ['h1', 'h2', 'h3', 'bold', 'italic', 'link', 'ul', 'ol']
```

Comment:
👍  Are there any other options in this list you considered or that we may want to include in the future? 

Also wondering if there will be any sanitization or parsing required when passing a string of html to the frontend via the API. There must be a trick to get it to render as HTML. Maybe we'll have to use the `dangerouslySetInnerHtml` method in React: https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml

Reply:
The API responses come back as HTML currently but it's fairly simple to modify if there's a better representation to use.

There are other options available [listed in the docs](http://docs.wagtail.io/en/v1.12/advanced_topics/customisation/page_editing_interface.html#limiting-features-in-a-rich-text-field) but these seemed like most of what an author would need.